docs/source/aryn_cloud/aryn_partitioning_service.rst

```diff
@@ -0,0 +1,148 @@
+Aryn Partitioning Service
+=========================
+
+.. toctree::
+   :caption: Aryn Cloud
+   :maxdepth: 1
+   :hidden:
+
+   ./aps_output.md
+
+You can use the Aryn Partitioning Service to segment PDF's into labeled bounding boxes identifying titles, tables, table rows and columns, images, and regular text. Bounding boxes are returned as JSON with their associated text for easy use. It leverages `Aryn's purpose-built AI model for document segmentation and labelling <https://huggingface.co/Aryn/deformable-detr>`_ that was trained using DocLayNet – an open source, human-annotated document layout segmentation dataset containing tens of thousands of pages from a broad variety of document sources.
+
+
+If you'd like to experiment with the service, you can use a UI to visualize how your documents are partitioned in the `Aryn Playground <https://www.play.aryn.ai/partitioning>`_. Also, you can view `a notebook with sample code using the service <https://github.com/aryn-ai/sycamore/blob/main/notebooks/ArynPartitionerPython.ipynb>`_ and `a notebook using the service with Langchain <https://github.com/aryn-ai/sycamore/blob/main/notebooks/ArynPartitionerWithLangchain.ipynb>`_.
```

Comment:
should it say "you can use the UI in the Playground<...> to visualize how your document will be parititoned"

Reply:
Done.